package.json

```diff
@@ -2,7 +2,7 @@
   "name": "template-registry-api",
   "version": "0.5.0",
   "dependencies": {
-    "@adobe/aio-lib-console": "^5.1.0",
+    "@adobe/aio-lib-console": "^5.2.1",
```

Comment:
We should revert this now and update it as a chore when the new version (5.3.0) is released so that tests can run

Reply:
Got it. Should this be done in a separate PR or a chore commit in this PR?

Reply:
Separate chore PR (after aio-lib-console is released) would be preferred

Reply:
Although it's not entirely necessary to make a PR updating the version for that. The new aio-lib-console version will get picked up on deploy since it's only a minor update (5.3.0) and we're using the caret `^` here in our package.json (and we don't use package-lock.json)

Reply:
Makes sense. Thanks!